client/static/keycloak/keycloak.json

```diff
@@ -2,9 +2,9 @@
   "realm": "nest",
   "auth-server-url": "https://dev-sso.pathfinder.gov.bc.ca/auth",
   "ssl-required": "external",
-  "resource": "namex-TEST",
+  "resource": "namex-DEV",
```

Comment:
Should this file be changed from test to dev keycloak? Does this get overridden in the server environments (in which case it doesn't matter)?